data/json/overmap/overmap_special/specials.json

```diff
@@ -3503,6 +3503,18 @@
     "occurrences": [ 0, 2 ],
     "flags": [ "CLASSIC", "URBAN", "MAN_MADE" ]
   },
+  {
+    "type": "overmap_special",
+    "id": "solar_farm",
+    "overmaps": [ { "point": [ 0, 0, 0 ], "overmap": "solar_farm_north" } ],
+    "connections": [ { "point": [ 0, 1, 0 ], "terrain": "road" } ],
+    "locations": [ "land" ],
+    "city_distance": [ -1, 2 ],
+    "city_sizes": [ 4, -1 ],
+    "occurrences": [ 0, 4 ],
```

Comment:
```suggestion
    "occurrences": [ 0, 2 ],
```

Reply:
Given the size of our overmaps this is probably still too common but it's a good number for testing.